sale_order_general_discount/models/product_product.py

```diff
@@ -6,10 +6,8 @@
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
-    general_discount_apply = fields.Boolean(
-        string="Apply general discount",
-        default=True,
-        required=True,
-        help="If this checkbox is ticked, it means changing general discount on sale order "
-        "will impact sale order lines with this related product.",
+    bypass_general_discount = fields.Boolean(
```

Comment:
Shouldn't that change need migration script ?

Reply:
@rousseldenis 
I don't think it's needed.
I made this change from a `cherry-pick` from another PR and  the `general_discount_apply` field does not exist in previous version of this module.